renderer/src/pages/Dashboard.tsx

```diff
@@ -0,0 +1,126 @@
+import { useEffect, useState, useCallback } from 'react'
+import {
+  getAllActivities, stopSaturnNode, startSaturnNode,
+  setFilAddress, getFilAddress,
+  getTotalEarnings, getTotalJobsCompleted
+} from '../lib/station-config'
+import { ActivityEventMessage } from '../typings'
+import ActivityLog from '../components/ActivityLog'
+import HeaderBackgroundImage from '../assets/img/header.png'
+import WalletIcon from '../assets/img/wallet.svg'
+import { useNavigate } from 'react-router-dom'
+
+const Dashboard = (): JSX.Element => {
+  const navigate = useNavigate()
+
+  const [address, setAddress] = useState<string | undefined>()
+  const [totalJobs, setTotalJobs] = useState<number>(0)
+  const [totalEarnings, setTotalEarnigs] = useState<number>(0)
+  const [activities, setActivities] = useState<ActivityEventMessage[]>([])
+  const shortAddress = (str: string | undefined) => str
+    ? str.substring(0, 4) + '...' + str.substring(str.length - 4, str.length)
+    : ''
+  const disconnect = async () => {
+    // TODO: move disconnect logic to backend
+    await stopSaturnNode()
+    await setFilAddress('').then(
+      () => { setAddress(undefined); navigate('/', { replace: true }) }
```

Comment:
```suggestion
      () => { setAddress(undefined); navigate('/wallet', { replace: true }) }
```

This shouldn't go back to onboarding

Reply:
Also, could you please refactor not to use `.then()`?